### PR TITLE
feat: integrate tags in admin panel

### LINKED ISSUE #406: feat(admin): integrate tag with admin panel
In our use case we have two types of tags:
   -> admin tags, which are used by team members or exec(CC,TC) to mark the progress, improvements of an article.
   -> public tags, which are visible to the general public.
   
Note: while working make sure to add "tag.admin" role in firebase user.

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixed #406 

### Description
image
image

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by 120EE0692** [comment]
@Shurtu-gal when ever making some changes related to UI make sure to add some screen shorts.

**Comment by 120EE0692** [comment]
fix #406 in this branch and rename the PR title appropriately.

**Comment by Shurtu-gal** [comment]
> fix #406 in this branch and rename the PR title appropriately.

Done

**Comment by 120EE0692** [comment]
@Shurtu-gal integrate auto complete of tags as well.

**Comment by 120EE0692** [comment]
@Shurtu-gal After adding or removing a tag, if user click on the update button it only show the tags which were initially present in them, look into this issue and fix.

**Comment by Shurtu-gal** [comment]
> @Shurtu-gal After adding or removing a tag, if user click on the update button it only show the tags which were initially present in them, look into this issue and fix.

Done

File: client/src/components/admin_v2/Common/FeaturedMedia.jsx [hunk 1]
```diff
@@ -4,13 +4,12 @@ import React, { useEffect, useState } from 'react';
...
-// media imports
-import SharedMedia, { thumb, thumbInner } from './sharedMedia';
-import imagekit from '../../../utils/imagekit';
-
 // graphql
 import { GraphClient } from '../../../config/ApolloClient';
 import addMedia from '../../../graphql/mutations/media/addMedia';
+import imagekit from '../../../utils/imageKit';
+// media imports
+import SharedMedia, { thumb, thumbInner } from './sharedMedia';
```
Comment (120EE0692) [inline client/src/components/admin_v2/Common/FeaturedMedia.jsx:12]: @Shurtu-gal the commit message made for these changes are not relevant.

File: client/src/pages/admin_v2/index.jsx [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 
 import Paperbase from '../../components/admin_v2/Marginals/Marginals';
-import Custom500 from '../500.jsx';
-
 import getAccess from '../../utils/getAccess';
+import Custom500 from '../500.jsx';
```
Comment (120EE0692) [inline client/src/pages/admin_v2/index.jsx:5]: @Shurtu-gal here as well the commit message makes no sense with the changes made.

**Review by 120EE0692 — APPROVED** [review]

**Pull request merged**